docs/configuration/integrations/gitlab-enterprise.md

```diff
@@ -1,35 +1,56 @@
 # GitLab Enterprise
 
-## GitLab Application
+Follow the instructions below to set up the Codacy integration with GitLab Enterprise:
 
-Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+1.  Open `<gitlab enterprise url>/profile/applications`, where `<gitlab enterprise url>` is the URL of your GitLab Enterprise instance.
 
-1.  Create a new application in `<gitlabEnterprise url>/profile/applications`, where `<gitlabEnterprise url>` is the URL of your own GitLab Enterprise instance, pointing to your local Codacy instance URL with `api`, `read user` and `read repository` scopes.
+2.  Fill in the fields to register your Codacy instance on GitLab:
 
-    You'll need to add the following "Redirect URI", making sure to update both your protocol to use either HTTP or HTTPS and your hostname as well. Keep in mind that this field is case sensitive.
+    -   **Name:** Name of the application. For example, `Codacy`.
 
-    ```
-    https://codacy.example.com/login/GitLab
-    https://codacy.example.com/add/addPermissions/GitLab
-    https://codacy.example.com/add/addProvider/GitLab
-    https://codacy.example.com/add/addService/GitLab
-    ```
+    -   **Redirect URI:** Copy the URLs below, updating the HTTP protocol and hostname to the correct values of your Codacy instance. This field is case sensitive.
+
+        ```text
+        https://<codacy hostname>/login/GitLab
+        https://<codacy hostname>/add/addProvider/GitLab
+        https://<codacy hostname>/add/addService/GitLab
+        https://<codacy hostname>/add/addPermissions/GitLab
+        ```
+
+    -   **Scopes:** Enable the scopes:
+    
+        - `api`
+        - `read_user`
+        - `read_repository`
```

Comment:
@jllopes Are these the correct configurations? They do not match what we had for Doplicated:

https://support.codacy.com/hc/en-us/articles/207994975-Configuring-GitLab-Enterprise-with-Codacy-Self-hosted

I also noticed that the configurations that existed before this merge used to match the Doplicated ones:

https://github.com/codacy/chart/commit/0dd82a1dcbab7e58a8a57633dda06f61d82f22c5#diff-dc8c06a3b4b1ffac2c62498bb26e0739R3

Reply:
Hey @pauloribeiro-codacy. Good catch, the URL's should have GitLabEnterprise and not GitLab, and the same for the yaml suggestion

Reply:
Also, it seems that the OpenID scope is also required